From 9a1269f796dcd7c3550c17dc06a0f9cf6e56c97e Mon Sep 17 00:00:00 2001
From: Kovacsics Robert <kovirobi@gmail.com>
Date: Sat, 3 Jan 2026 14:44:41 +0000
Subject: [PATCH] Fix pre-C23 any type headers

Pre C23 a function with no arguments (instead of e.g. `void
destroy(void)`) was any type not no args.
---
 src/pasystray.h | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/pasystray.h b/src/pasystray.h
index e131b8d..29c373f 100644
--- a/src/pasystray.h
+++ b/src/pasystray.h
@@ -22,8 +22,9 @@
 #ifndef PASYSTRAY_PASYSTRAY_H
 #define PASYSTRAY_PASYSTRAY_H
 
-void init();
+typedef struct settings_t_ settings_t;
+void init(settings_t* settings);
 void quit();
-void destroy();
+void destroy(settings_t* settings);
 
 #endif /* PASYSTRAY_PASYSTRAY_H */
-- 
2.51.2

